documentation/docs/getting_started/docker.md

```diff
@@ -4,20 +4,117 @@ keywords:
 - docker
 ---
 
-# Docker
+# Setting up `Hornet` with `Chronicle` as INX plugin for permanent data storage and Tangle and Ledger live analytics
 
-## Usage
+This is a guide to set up a `Hornet` node together with `Chronicle` as an INX plugin to persist Tangle data without pruning (make sure to maintain sufficient hard disk storage), gather time series analytics data from the Tangle and the UTXO ledger, and display such data in meaningful ways using Grafana. 
 
-The easiest way to start Chronicle is by using our supplied Dockerfile.
+Once you've cloned the repository from [Github](https://github.com/iotaledger/inx-chronicle.git), switch to the `docker` folder:
+```sh
+cd inx-chronicle/docker
+```
+
+## 1. Create the necessary directories for all `Hornet` and `Chronicle` databases
+
+```sh
+sudo ./create_dirs.sh
+```
+
+## 2. Configure `Hornet` by editing the `docker-compose.yml` file
+
+```sh
+<your-favorite-text-editor> docker-compose.yml
+```
+
+### Head to `Hornet`s CLI section:
+
+```yml
+hornet:
+    command:
+```
+
+### (Optional) Select another network (default=Shimmer) by uncommenting one of the `config*.json` lines as shown below:
+
+```yml
+- "-c"
+# - "config_testnet.json"
+# - "config_alphanet.json"
+```
+### Configure peers for Hornet:
+
+The most convenient way is to use autopeering. Hornet will try to find its peers automatically and learn about more and more nodes in the particular network, and switch between them regularly. 
+
+```yml
+- "--p2p.autopeering.enabled=true"
+```
+However, this also reveals the existence of your node to everyone else in the network, which you may not want to. In that case, you should set up manual peering only (but you can also use it additionally). This is how you set it up:
```

Comment:
Ending a sentence with a preposition?? Not in my documentation!

Reply:
oh shit...right